lib/generators/task/templates/deploy.txt.erb

```diff
@@ -8,5 +8,5 @@ namespace :after_party do
     # Update task as completed.  If you remove the line below, the task will
     # run with every deploy (or every time you call after_party:run).
     AfterParty::TaskRecord.create version: '<%= timestamp %>'
-  end  # task :<%= file_name %>
-end  # namespace :after_party
+  end # task :<%= file_name %>
```

Comment:
Hey I noticed here that you deleted a white space and I think that could seriously impair the functioning of our application. Let's write some specs to test if this is even feasible.

Reply:
Just curious, how could deleting this whitespace _"seriously impair the functioning of our application"_? Removing the whitespace here should have no impact, unless I really missing something.

Reply:
You're correct. This was my awful attempt at humor.